tests/Oryx.Tests.Common/TestTempDirTestFixture.cs

```diff
@@ -6,7 +6,7 @@
 using System;
 using System.IO;
 
-namespace Microsoft.Oryx.Tests.Common
+namespace Microsoft.Oryx.Detector.Tests
```

Comment:
why the rename?

Reply:
This was by mistake. I will revert it actually.